client/dashboard/data/types.ts

```diff
@@ -187,3 +187,15 @@ export interface DefensiveModeSettingsUpdate {
 	active: boolean;
 	ttl?: number;
 }
+
+export type SiteContent = {
```

Comment:
nit: Would it be better to call it `SiteResetContentSummary`? `SiteContent` feels a bit ambiguous.